test/cmd/admin.sh

```diff
@@ -299,3 +299,8 @@ os::cmd::expect_success_and_not_text "oc get clusterrolebindings/cluster-admins
 os::cmd::expect_success_and_not_text "oc get rolebindings/cluster-admin         --output-version=v1 --template='{{.subjects}}' -n default" 'cascaded-group'
 os::cmd::expect_success_and_not_text "oc get scc/restricted                     --output-version=v1 --template='{{.groups}}'"              'cascaded-group'
 echo "user-group-cascade: ok"
+
+
+os::cmd::expect_failure_and_text 'openshift ex diagnostics FakeDiagnostic AlsoMissing' 'No requested diagnostics are available: requested=FakeDiagnostic. AlsoMissing'
```

Comment:
Except parameter 2 to `os::cmd::expect_failure_and_text` is a regex, and `.` matches `,`, so it should work, no? (Cute!)